todo/src/reducers/index.js

```diff
@@ -0,0 +1,17 @@
+import { ADD_TODO, UPDATE_COMPLETED, DELETE_ITEM } from '../actions';
+
+export default (todos = [], action) => {
+  console.log('Action: ', action);
+  switch (action.type) {
+    case ADD_TODO:
+      return [...todos, action.payload];
+    case UPDATE_COMPLETED:
+      return todos.map(todo => {
+        return todo.id === action.payload.id ? { ...todo, completed: !todo.completed } : todo;
```

Comment:
no need for brackets and return statement here, just remove them and use arrow function one line return syntax.  Feel free to split it up on multiple lines as well. It won't break the code.